scripts/entrypoint.sh

```diff
@@ -35,6 +36,16 @@ if [[ ${RESTORE_BACKUP^^} == TRUE && -z ${BACKUP_NAME} ]]; then
   exit 1
 fi
 
+if [[ ${BACKUPS^^} == TRUE ]] && [[ ! -z ${CRON_SCHEDULE}  ]] && [[ $(id -u) == 0 ]]; then
+  echo "Starting cron job scheduler" && crond
+  echo "Database backups will be scheduled to run at ${CRON_SCHEDULE}. Check https://crontab.guru/ for schedule expression details"
+fi
+
+function cron_schedule() {
+    CRON_CONFIGURATION="${CRON_SCHEDULE} /usr/local/scripts/base_backup.sh >> /var/log/pg-backups.log"
```

Comment:
Does this not need to be going to standard out?

Reply:
Done